staging/src/k8s.io/apimachinery/pkg/api/meta/interfaces.go

```diff
@@ -75,6 +75,9 @@ type MetadataAccessor interface {
 	Annotations(obj runtime.Object) (map[string]string, error)
 	SetAnnotations(obj runtime.Object, annotations map[string]string) error
 
+	Continue(obj runtime.Object) (string, error)
+	SetContinue(obj runtime.Object, c string) error
```

Comment:
doesn't this belong in ListMetaAccessor, not MetadataAccessor?

Reply:
also, surprised this didn't complain that this wasn't implemented in *ObjectMeta, genericAccessor, etc

Reply:
There are three accessors.  A generic one (that has all methods and returns an error if it doesn't support it), and then one specific one for lists and one for metadata.